### PR TITLE
Update SPARQL->FQL tests to expect new query syntax

### PR DESCRIPTION
**Pull request by cap10morgan**

### CODE REVIEW

**Comment by cap10morgan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #636** <a href="https://app.graphite.dev/github/pr/fluree/db/636" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/fluree/db/636?utm_source=stack-comment).

File: test/fluree/db/query/sparql_test.cljc [hunk 1]
```diff
@@ -9,587 +9,602 @@
...
+          (testing "simple triple"
+            (let [query "SELECT ?person
                  WHERE {?person person:handle \"jdoe\".}"
-          {:keys [where]} (sparql/->fql query)]
-      (is (= [["?person" "person:handle" "jdoe"]]
-             where))))
-  (testing "multi clause"
-    (let [query "SELECT ?person ?nums
+                  {:keys [where]} (sparql/->fql query)]
+              (is (= '[{"@id"           "?person"
```
Comment (mpoffald) [inline test/fluree/db/query/sparql_test.cljc:88]: Super-trivial, but I don't think you need the quote here

**Review by mpoffald — APPROVED** [review]
🍬

File: test/fluree/db/query/sparql_test.cljc (hunk 1 above)
Comment (cap10morgan) [reply test/fluree/db/query/sparql_test.cljc:88]: Oh, nope. At one point I thought it was going to output symbols now, but it doesn't. Thanks!

**Comment by cap10morgan** [comment]
### Merge activity

* **Nov 1, 4:50 PM**: @cap10morgan started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/fluree/db/636).

**Pull request merged**